### ISSUE TITLE
Add a print note function

### ISSUE THREAD

**Issue by dhavelin** [dendronhq/dendron#1623]
## Please select if your request is either something new or an enhancement ##

- [ ] Enhancement of an existing Feature.
- [x] Request of a new feature.

## Please select the area your request applies to. (Multiple selections are possible) ##

- [ ] Onboard. Initial vault setup and import/export pods
- [ ] Create. Note creation, lookup, snippets and templates
- [ ] Retrieve. Backlinks, references, graph view
- [ ] Structure. Refactoring, multi-vault and schemas
- [ ] Publish. Sharing your repo with the world
- [x] Misc (Choose this if you're not sure)

## Is your feature request related to a problem? Please describe ##

I store recipes in my Dendron vault. I wanted to print one to bring to the kitchen but discovered that neither Dendron nor VSCode provides a print function.

## Describe the solution you'd like ##

I probably always want to print the preview version of the note. When a preview view is active, I'd like to be able to choose a Dendron:Print command (mapped to Cmd-P?) that sends the content to a printer.

## Describe alternatives you've considered ##

I can, of course, export to HTML via a pod, incorporate the HTML into a webpage and print using a browser. It is an almost universal convention across applications, however, to have a simple print command whenever there is user-editable content (and even with non-user editable content). VSCode is the only example I can think of that doesn't support printing, and that should have it too.

Various extensions to VSCode offer the prospect of being able to print, typically by running some kind of server. It seems quite a heavyweight solution and my laptop already runs quite hot. I didn't test them. I also avoid plugins and extensions generally to reduce the maintenance and troubleshooting burden. Printing seems like basic functionality that would ideally come built in.

**Comment by jonathanyeung** [comment]
Thanks @dhavelin for the suggestion.  

Implementation wise, I think this one would be a bit more complex to support - we'd need OS specific logic for each platform.

Perhaps a simpler solution would be for us to introduce a command like "Show Preview in Browser", which could quickly render a preview page in the browser in one-click (as opposed to the current flow of configure pod export, run pod export, copy, paste, open .html in browser, etc.).  Then, one could utilize the print capabilities of the browser, as you suggested.

**Comment by dhavelin** [comment]
I agree, @jonathanyeung, "Show Preview in Browser" is a very sensible approach. Thank you!

**Comment by sidwellr** [comment]
I would like to see this feature added as well. "Show Preview in Browser" to use the browser's print function would be fine.
